server/channels/api4/file_test.go

```diff
@@ -1147,7 +1165,8 @@ func TestGetPublicFile(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, th.cleanupTestFile(fileInfo))
 
-	th.cleanupTestFile(info)
+	err = th.cleanupTestFile(info)
+	require.NoError(t, err)
```

Comment:
This code cleans up the test files 2 times, leading to an error. Please remove one call to `cleanupTestFile`